tox.ini

```diff
@@ -0,0 +1,8 @@
+[tox]
+envlist = py27, py35, py36, py37
+
+[testenv]
+commands = py.test -v -x
+deps = pytest
```

Comment:
I haven't used tox before so I am not totally familiar with how it installs things, but are `nbconvert` and `scons` necessary to list here since they will also be installed via `install_requires`?

Reply:
You're right, it wasn't necessary, just pytest is require